<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addCities</class>
 <widget class="QWidget" name="addCities">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="MainWidget">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="3">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Cities</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="addCity_pushButton">
           <property name="text">
            <string>Add New city</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="editCity_pushButton">
           <property name="text">
            <string>Edit Selected</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteCity_pushButton">
           <property name="text">
            <string>Delete Selected</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="write_Button">
           <property name="text">
            <string>Write Cities to File</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="close_pushButton">
           <property name="text">
            <string>Close Settigns</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="3">
        <widget class="QListWidget" name="cities_listWidget"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="addNewCityWidget">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="3" column="3">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Food</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="add_foodTable"/>
         </item>
         <item>
          <widget class="QPushButton" name="add_addFood">
           <property name="text">
            <string>Add New Food</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="add_deleteFood">
           <property name="text">
            <string>Delete Selection</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Add New City</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" rowspan="2">
        <widget class="QStackedWidget" name="add_Stack">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page_3">
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>City Name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="add_setName"/>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Parent Country Name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="add_setCountry"/>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="add_addDistances">
               <property name="text">
                <string>Add Distances</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="add_Cancel">
               <property name="text">
                <string>Cancel</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_4">
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="1" column="0">
            <widget class="QTableWidget" name="distance_Table"/>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="pushButton_3">
             <property name="text">
              <string>Back</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="save_City">
             <property name="text">
              <string>Save Changes</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Distances Between Cities</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="EditCities">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="9">
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Food</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="edit_foodTable"/>
         </item>
         <item>
          <widget class="QPushButton" name="edit_addFood">
           <property name="text">
            <string>Add New Food</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="edit_deleteFood">
           <property name="text">
            <string>Delete Selection</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="8">
        <widget class="QStackedWidget" name="editCitiesStack">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="1" column="0">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>City Name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="editCities_namelineEdit"/>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Parent Country Name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="editCities_countrylineEdit"/>
             </item>
             <item>
              <widget class="QPushButton" name="edit_editDistance">
               <property name="text">
                <string>Edit Distances</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="edit_Save">
               <property name="text">
                <string>Apply Changes</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="edit_Cancel">
               <property name="text">
                <string>Cancel</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Edit City</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2">
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="2" column="0">
            <widget class="QPushButton" name="editCities_distBack">
             <property name="text">
              <string>Back</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QTableWidget" name="edit_Table"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Distances Between Cities</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
